kong/llm/plugin/base.lua

```diff
@@ -93,6 +94,7 @@ end
 
 function MetaPlugin:retry(sub_plugin, conf)
   run_stage(STAGES.REQ_TRANSFORMATION, sub_plugin, conf)
+  balancer.recreate_request()
```

Comment:
let's move this responsiblity to PDK, for example, this should be called by
`kong.request.set_path` and when it's balancer phase.

Reply:
Let's do it in another PR.